corpus/basic.txt

```diff
@@ -8,27 +8,112 @@ Empty docblock
 (document)
 
 =====================
-Docblock w/ no tags
+Docblock w/o tags
 =====================
 /**
- * Just a comment
+ * Summary text
```

Comment:
@claytonrcarter I wonder if we should introduce a `(description (summary)` whenever we find a description with more than 1 paragraph. It's mentioned in phpDocs documentation here: https://docs.phpdoc.org/3.0/guide/guides/docblocks.html But this might be difficult to parse and introduce more questions than it's worth (what if there's only 1 paragraph? Is it `text` or `summary`?).

We also don't include separating empty lines between paragraphs. Something that could make problems if someone wants to inject markdown on top of phpdoc.

Reply:
On second thought summary detection is better to be covered on the query side.

Reply:
I'm not opposed, but I'm inclined to wait for something like this until the current parse behavior is insufficient for someone's IRL use case.